mail_tracking_mailgun/models/res_partner.py

```diff
@@ -31,28 +32,33 @@ def _email_bounced_set(self):
     def _mailgun_values(self):
         icp = self.env['ir.config_parameter']
         api_key = icp.get_param('mailgun.apikey')
-        api_url = icp.get_param('mailgun.api_url')
-        domain = icp.get_param('mailgun.mydomain')
+        if not api_key:
+            raise UserError(_('There is no Mailgun API!'))
```

Comment:
There's an API, just not the key 😉

Wouldn't this one be a `ValidationError`, since its purpose is to validate that the requirements are present? 🤔